services/minespace-web/src/components/dashboard/mine/MineDashboard.js

```diff
@@ -139,6 +140,9 @@ export class MineDashboard extends Component {
                   <Tabs.TabPane tab="Permits" key="permits">
                     <Permits mine={mine} match={this.props.match} />
                   </Tabs.TabPane>
+                  <Tabs.TabPane tab="Notices of Departure" key="nods">
```

Comment:
Awesome!  Thanks Ben.  I had spoken to Hitankar and was looking at the Authorization wrapper for this, but wasn't able to achieve the desired result with that.  This will do nicely!